src/main/java/com/pmm/budgetcalculator/expensing/repositories/Expense.java

```diff
@@ -26,6 +28,7 @@ public class Expense extends BaseEntity {
     private Date expenseTime;
     private BigDecimal expenseAmount;
     private String expensePlace;
-
+    @OneToMany(mappedBy = "expense", cascade = CascadeType.ALL)
+    private List<ExpenseType> expenseTypeList;
```

Comment:
Generally we don't name Lists by list suffix. ExpenseTypes is good enough we can see that this is list thanks to java verbose syntax